### PR TITLE
Working branch

### PR DESCRIPTION
**Pull request by ljte**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+    - "3.8"
+
+install: pip install -r requirements.txt
```
Comment (inauros) [inline .travis.yml]: The "install" section should include instructions for installing the application using setup.py. Dependencies should be installed as part of this process, without explicitly using pip.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+script:
+  - pylint department-app/tests
+  - coverage run department-app/tests
+after-success:
+  - coveralls
```
Comment (inauros) [inline .travis.yml:13]: Please, add an ability to send email notifications.

File: README.md [hunk 3]
```diff
@@ -20,3 +20,7 @@ The user of the app will be able to:
 
 ## Installation
 coming soon
+
+
+
+[![Coverage Status](https://coveralls.io/repos/github/ljte/lab-project/badge.svg?branch=working_branch)](https://coveralls.io/github/ljte/lab-project?branch=working_branch)
```
Comment (inauros) [inline README.md]: Please, add the [Build Status] section as well.

Comment (inauros) [inline README.md]: When you create a pull request, your [Build Status] and [Coverage Status] should point to the master branch.

File: department-app/tests/__main__.py [hunk 4]
```diff
@@ -0,0 +1,3 @@
+"""tests for the app"""
+
+print("hello, coveralls")
```
Comment (inauros) [inline department-app/tests/__main__.py]: This file is redundant and can be removed.

**Review by inauros — CHANGES_REQUESTED** [review]
1. Please, add .coverage file to the .gitignore list
2. Please, update the setup.py (install_requires) to parse dependencies from the requirements.txt file using the parse_requirements() function
3. Please, add the .coveragerc file to the root of the project and refactor it to exclude tests from the coverage report
4. Please, move the [Build Status] and [Coverage Status] sections at the top of the Readme.md file
5. Please, add .coverage to the .gitignore list

File: department-app/tests/__main__.py (hunk 4 above)
Comment (ljte) [reply department-app/tests/__main__.py]: done

File: README.md (hunk 3 above)
Comment (ljte) [reply README.md]: done

Comment (ljte) [reply README.md]: done

File: .travis.yml (hunk 2 above)
Comment (ljte) [reply .travis.yml:13]: done

File: .travis.yml (hunk 1 above)
Comment (ljte) [reply .travis.yml]: done

File: .coveragerc [hunk 5]
```diff
@@ -0,0 +1,3 @@
+[report]
```
Comment (inauros) [inline .coveragerc]: I would recommend [run] instead of [report]

File: .travis.yml (hunk 2 above)
Comment (inauros) [reply .travis.yml:13]: Recipient list should be specified

File: README.md [hunk 6]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.com/ljte/lab-project.svg?branch=master)](https://travis-ci.com/ljte/lab-project) [![Coverage Status](https://coveralls.io/repos/github/ljte/lab-project/badge.svg?branch=master)](https://coveralls.io/github/ljte/lab-project?branch=working_branch)
+
```
Comment (inauros) [inline README.md:2]: (https://coveralls.io/github/ljte/lab-project?branch=working_branch) - should be master

**Review by inauros — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (ljte) [reply README.md:2]: done

File: .coveragerc (hunk 5 above)
Comment (ljte) [reply .coveragerc]: done

**Review by inauros — APPROVED** [review]

**Pull request merged**